onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2645,6 +2645,18 @@ def aten_upsample_nearest3d(
     return _aten_upsample_output_size(self, size, "nearest", "asymmetric")
 
 
+@torch_op("aten::upsample_nearest3d.vec", trace_only=True)
+def aten_upsample_nearest3d_vec(
+    input: TReal,
+    output_size: INT64,
```

Comment:
nit: should we use same name for parameter in previous and this function?

Reply:
I am following the function signature: https://github.com/pytorch/pytorch/blob/52c917b0bab712ef103d8e0f8eba687819e15070/aten/src/ATen/native/native_functions.yaml#L12570